java/client/src/main/java/glide/api/BaseClient.java

```diff
@@ -53,7 +53,9 @@
 import glide.api.commands.SetBaseCommands;
 import glide.api.commands.SortedSetBaseCommands;
 import glide.api.commands.StringCommands;
+import glide.api.models.Script;
 import glide.api.models.commands.ExpireOptions;
+import glide.api.models.commands.ScriptOptions;
```

Comment:
Why not to import the `glide.api.models.commands` once, instead of importing the `ScriptOptions`, `ExpireOptions` and ...?